File: Java-Solutions/find-longest-word-in-string.java
    public static int longestWordLength(String str) {
        String[] stringArray = str.split(" ");
        int greatestNum = 0;
        for (String word : stringArray) {
            if (word.length() > greatestNum) {
                greatestNum = word.length();
            }
        }
        return greatestNum;
    }

File: JavaScript-Solutions/find-longest-word-in-string.js

function findLongestWord(str) {
  var stringArray = str.split(" ");
  var greatestNum = 0;
  for (var i = 0; i < stringArray.length; i++) {
    if (stringArray[i].length > greatestNum) {
      greatestNum = stringArray[i].length;
    }
  }
  return greatestNum;
}
